recipes/r-qgisprocess/meta.yaml

```
@@ -0,0 +1,106 @@
+{% set version = '0.1.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-qgisprocess
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/qgisprocess_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/qgisprocess/qgisprocess_{{ version }}.tar.gz
+  sha256: 593a4a52131892306aabe9b298d33057c59d681fa0301fba8d0e136dbe55ae1c
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+
+  host:
+    - r-base
+    - r-assertthat
+    - r-glue
+    - r-jsonlite
+    - r-processx >=3.5.2
+    - r-rappdirs
+    - r-rlang
+    - r-stringr
+    - r-tibble
+    - r-vctrs
+    - r-withr
+
+  run:
+    - r-base
+    - r-assertthat
+    - r-glue
+    - r-jsonlite
+    - r-processx >=3.5.2
+    - r-rappdirs
+    - r-rlang
+    - r-stringr
+    - r-tibble
+    - r-vctrs
+    - r-withr
+
+test:
+  commands:
+    - $R -e "library('qgisprocess')"           # [not win]
+    - "\"%R%\" -e \"library('qgisprocess')\""  # [win]
+
+about:
+  home: https://r-spatial.github.io/qgisprocess/, https://github.com/r-spatial/qgisprocess
+  license: GPL-3.0-only
```

DESCRIPTION shows `GPL (>= 3)`

```suggestion
  license: GPL-3.0-or-later
```

---

Thanks, done!